From 147c496d5b2a33b7534ac66dbfb7705ae7ccb7bd Mon Sep 17 00:00:00 2001
From: Shane Bradley <sbradley@redhat.com>
Date: Wed, 11 Mar 2015 09:55:06 -0400
Subject: [PATCH] [cluster] crm_report fails to run because dir already exists

Do not create directory for the destination path used by crm_report.

Resolves: rhbz#1200526

Signed-off-by: Shane Bradley <sbradley@redhat.com>
Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/cluster.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sos/plugins/cluster.py b/sos/plugins/cluster.py
index bd2cf8a..8dd9a4d 100644
--- a/sos/plugins/cluster.py
+++ b/sos/plugins/cluster.py
@@ -111,7 +111,7 @@ class Cluster(Plugin, RedHatPlugin):
                     "crm_from parameter '%s' is not a valid date: using "
                     "default" % self.get_option('crm_from'))
 
-        crm_dest = self.get_cmd_output_path(name='crm_report')
+        crm_dest = self.get_cmd_output_path(name='crm_report', make=False)
         self.add_cmd_output('crm_report -S -d --dest %s --from "%s"'
                             % (crm_dest, crm_from))
 
-- 
1.7.1

